python-lib/spacy_tokenizer.py

```diff
@@ -186,6 +192,53 @@ def __init__(
                 f"enable_pipe_components and disable_pipe_components are both non-empty. Please give either components to enable, or components to disable."
             )
 
+    def _get_error_message_lemmatization(self, language: AnyStr) -> AnyStr:
+        """Return the error message to display when the lemmatization cannot be applied"""
+        if language in SPACY_LANGUAGE_MODELS_LEMMATIZATION:
+            # 'Russian','Polish' without a pretrained model
+            return f"The language '{language}' is not available for Lemmatization without loading a pre-trained model. Set use_models to True to lemmatize in '{language}'"
+        else:
+            # Any unsupported language
+            return f"The language '{language}' is not available for Lemmatization. Uncheck the lemmatization option and re-run the recipe."
+
+    def _get_components_to_activate(self, language: AnyStr) -> List[AnyStr]:
```

Comment:
Need to be more precise in the function name
```suggestion
    def _get_components_to_activate_lemmatization(self, language: AnyStr) -> List[AnyStr]:
```